chapter_08.md

```diff
@@ -1,19 +1,13 @@
 Cloud Integration
 =================
 
-Now as we have a pretty decent functional test suite, let's scale this up and run everything on the cloud using more capabilities to ensure that our example applications works cross functional. You can pick between the cloud vendor of your choice.
+Let's scale this up and run everything on the cloud using more capabilities to ensure that our example applications works cross functional. You can pick between the cloud vendor of your choice.
 
-# [Perfecto](https://www.perfecto.io/)
+<details>
 
-The objectives are:
-
-1. Export Perfecto credentials as environment variables
-2. Create a separate config file to run tests locally or in Perfecto
-3. Modify your NPM scripts to be able to run tests locally or in Perfecto
-5. Add 3 more browser capabilities to run your tests
-6. Make sure all your tests run in the same build
+<summary>Sauce Labs</summary>
```

Comment:
@christian-bromann + @erwinheitzman 

Why use Sauce, they don't sponsor us anymore, can't we go for BS or LT?

Reply:
We had this section even before the sponsorship. I think there is no reason to remove them now. I don't mind though adding more vendors so folks can use whatever is already used in their organisation.

Reply:
I have added Browserstack for now, we can add others later.